### ISSUE TITLE
Inspection Report - Better Expose what's possible to do

### ISSUE THREAD

**Issue by franck-boullier** [unee-t/frontend#486]
# The problem:

In the current design (as per figma), when a user creates an Inspection Report he/she has to scroll down to see that it's possible to create
- Cases
- Inventory Items
- Rooms
- Add remarks and comments

![image](https://user-images.githubusercontent.com/31331637/45140816-a2fec480-b1e6-11e8-9c6a-e140c5df8064.png)

# What we currently have in Production:

In the current production we have implemented a slightly different design which was discussed [here](https://github.com/unee-t/frontend/pull/440#issuecomment-414352587)
Below is a summary of what was said.

## Detailed list of the suggested change:

- Move the __'Additional Comments'__ section right after the __'Attach photos'__ section
- Rename __'Additional Comments'__ to __'Remarks and Comments'__

## Rationale for the change"
better differentiate 
- __'remarks'__: there is no expected fix, this is just captured for future reference and to make sure that the condition of the unit is properly documented.
- __'cases'__: a fix is expected ---> they are more complex/time consuming to create

## The current design in Production:
![image](https://user-images.githubusercontent.com/31331637/45141069-6ed7d380-b1e7-11e8-8eda-9e566294ce9d.png)

# Things that need to be improved:

We want to avoid the situation where a user starts to create an Inspection Report, add pictures and Remarks and Comments and ONLY THEN discovers that he/she has the ability to create a Room and/or create an Item.

How could we do that efficiently?

# Open Questions:

Invite user to collaborate is not a key functionality for the IR ---> should we move this further down?
Further reading: see https://github.com/unee-t/frontend/issues/471

# More Context:
The different roles for users in an Inspection Report are described in the Inspection Report project described [here](https://github.com/orgs/unee-t/projects).

**Comment by franck-boullier** [comment]
@nbiton update on were the design is going. This is still WIP for now. @kiatlim will post more in figma soon.

![image](https://user-images.githubusercontent.com/31331637/45389995-33f2f700-b650-11e8-8a01-246a2613456b.png) 

![image](https://user-images.githubusercontent.com/31331637/45390052-6c92d080-b650-11e8-8f2a-ee8a2d41ce6f.png)

**Comment by nbiton** [comment]
The "Cases" tab looks like it needs to be more carefully thought out. Showing all the cases at once is going to be very tricky to get done without it getting very confusing.
* "Cases for the overall unit" - pretty straightforward, not confusing. Adding a case creates a case under the report.
* "Cases from Rooms" - requires grouping by room (?) or showing which room each case belongs to. Adding a case requires specifying a room.
* "Cases from Inventory" - requires grouping by inventory items, which themselves can belongs to the entire report or a room, so it might require 2 levels of grouping (???). Adding a case requires specifying an inventory item (I guess that can be done in a single selection, but that would still need to group inventory items by room, just for the selection view).

In short, going down this path seems like a rabbit hole of complexity.

**Comment by kiatlim** [comment]
@nbiton totally agree, they're just preliminary screens fresh off discussion with @franck-boullier yesterday afternoon. Will be testing it out the flow and see if it all works.

**Comment by franck-boullier** [comment]
# Ideas:

- "Cases from Rooms": maybe we can just list the cases but not allow the user to add cases directly 
- "Cases from Inventory": maybe we can just list the cases but not allow the user to add cases directly

---> that way we avoid complexity while creating a new case AND we have a tab with a clear overview of all the cases associated to that Inspection Report...

**Comment by franck-boullier** [comment]
@nbiton in the meantime we can probably implement the new design with 2 tabs "Overview" and "Cases" and omit the "Rooms" and "Inventory" dimensions/Objects ---> this would be an improvement compared to what we have now...

**Comment by nbiton** [comment]
For the Oct 10 milestone, we'll implement just the "Rooms" + cases under "Rooms"

**Comment by nbiton** [comment]
The tabbed layout was implemented but it has only "Overview" and "Cases".
I suggest we close this in favor of GH issues that focus on implementing "Rooms" and "Inventory" (items) specifically.
